userspace/libsinsp/sinsp.h

```diff
@@ -484,54 +484,6 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 	*/
 	threadinfo_map_t::ptr_t get_thread_ref(int64_t tid, bool query_os_if_not_found = false, bool lookup_only = true, bool main_thread = false);
 
-	/*!
-	  \brief Return the table with all the machine users.
-
-	  \return a hash table with the user ID (UID) as the key and the user
-	   information as the data.
-
-	  \note this call works with file captures as well, because the user
-	   table is stored in the trace files. In that case, the returned
-	   user list is the one of the machine where the capture happened.
-	*/
-	const unordered_map<uint32_t, scap_userinfo*>* get_userlist();
-
-	/*!
-	  \brief Lookup for user in the user table.
-
- 	  \return the \ref scap_userinfo object containing full user information,
- 	   if user not found, returns NULL.
-
- 	  \note this call works with file captures as well, because the user
-	   table is stored in the trace files. In that case, the returned
-	   user list is the one of the machine where the capture happened.
-	*/
-	scap_userinfo* get_user(uint32_t uid);
-
-	/*!
-	  \brief Return the table with all the machine user groups.
-
-	  \return a hash table with the group ID (GID) as the key and the group
-	   information as the data.
-
-	  \note this call works with file captures as well, because the group
-	   table is stored in the trace files. In that case, the returned
-	   user table is the one of the machine where the capture happened.
-	*/
-	const unordered_map<uint32_t, scap_groupinfo*>* get_grouplist();
-
-	/*!
-	  \brief Lookup for group in the group table.
-
-	  \return the \ref scap_groupinfo object containing full group information,
-	   if group not found, returns NULL.
-
- 	  \note this call works with file captures as well, because the group
-	   table is stored in the trace files. In that case, the returned
-	   group list is the one of the machine where the capture happened.
-	*/
-	scap_groupinfo* get_group(uint32_t gid);
-
```

Comment:
Why don't we leave these as proxy methods to usergroup_manager?

Reply:
Yeah, i just don't think they're used indeed. One can just call the user_group_manager.
I haven't got strong opiniong on that though.